### ISSUE TITLE
Holy Oration has an additive benefit on the base CDR

### ISSUE THREAD

**Issue by Niphyr** [MechanicalPriest/Salvation#58]
tl;dr Holy Oration has an additive benefit on the base CDR, the others are all multiplicative.

## Base CDR
FH/Heal - 6s
Renew - `(6 / 3)` = 2s
PoH - 6s

## LotN CDR
FH/Heal - `6 * (1 + 1/3)` = 8s
Renew - `(6 / 3) * (1 + 1/3)` = 2.67s
PoH - `6 * (1 + 1/3)` = 8s

## HO(5%) CDR
FH/Heal - `6 + (6 * 0.05)` = 6.3s
Renew - `(6 / 3) + ((6 / 3) * 0.05)` = 2.1s
PoH - `6 + (6 * 0.05)` = 6.3s

## LotN + HO(5%) CDR 
FH/Heal - `6 * (1 + 1/3) + (6 * 0.05)` = 8.3s
Renew - `(6 / 3) * (1 + 1/3) + ((6 / 3) * 0.05)` = 2.76s
PoH - `6 * (1 + 1/3) + (6 * 0.05)` = 8.3s

## Apothesosis CDR
FH/Heal - `6 * 3` = 24s
Renew - `(6 / 3) * 3` = 8s
PoH -`6 * 3` = 24s

## HO(5%) CDR + Apothesosis
FH/Heal - `6 * (1 + 1/3) * 3 + (6 * 0.05)` = 24.3s
Renew - `(6 / 3) * (1 + 1/3) * 3 + ((6 / 3) * 0.05)` =8.1s
PoH -`6 * (1 + 1/3) * 3 + (6 * 0.05)` = 24.3s

_Originally posted by @Niphyr in https://github.com/MechanicalPriest/Salvation/issues/6#issuecomment-700671540_

**Comment by Niphyr** [comment]
Turns out this wasn't a bug, but a feature.
